challenges/security/application-security.md

```diff
@@ -0,0 +1,270 @@
+# Summary
```

Comment:
Not sure what happened, but all my edits got lost somehow. I had made a lot of spelling and grammar fixes in a previous commit. I'll download this and do a pass separately.

Reply:
Sorry -- I must have force pushed over the changes when I rebased  at https://github.com/gravitational/careers/pull/66#event-7224472835 -- I typically try to pick them up.

I think I can probably fish them out of https://github.com/gravitational/careers/commit/952d8e7f53aa7c973a1ccd6488b0ae849ca62c70

Reply:
Reintroduced in https://github.com/gravitational/careers/pull/66/commits/66bf056ee02af1554320e364a53269a4b3ea685e